push_notifications/models.py

```diff
@@ -257,3 +256,29 @@ def send_message(self, message, **kwargs):
 		return webpush_send_message(
 			uri=self.registration_id, message=message, browser=self.browser,
 			auth=self.auth, p256dh=self.p256dh, application_id=self.application_id, **kwargs)
+
+
+class FCMDevice(GCMDevice):
```

Comment:
The migration for existing users doesn't make sense. You should instead be renaming the current model to GCP no?